connector/connect/client/jvm/src/test/scala/org/apache/spark/sql/application/ReplE2ESuite.scala

```diff
@@ -151,4 +152,32 @@ class ReplE2ESuite extends RemoteSparkSession with BeforeAndAfterEach {
     assertContains("Array[java.lang.Long] = Array(0L, 2L, 4L, 6L, 8L)", output)
   }
 
+  test("Client-side JAR") {
+    // scalastyle:off classforname line.size.limit
+    val sparkHome = IntegrationTestUtils.sparkHome
+    val testJar = Paths
+      .get(s"$sparkHome/connector/connect/client/jvm/src/test/resources/TestHelloV2.jar")
```

Comment:
Looks fine but for doubly sure, does it need Scala 2.13 jar too, @LuciferYang ?

Reply:
Yes, should wait https://github.com/apache/spark/pull/41852

Reply:
Thanks! I've updated the code to account for the scala version